### ISSUE TITLE
Add test cases for jar: scheme

### ISSUE THREAD

**Issue by hillbrad** [w3c/webappsec#479]
https://jsfiddle.net/6yujhzyp/1/

**Comment by annevk** [comment]
Note that per Fetch this is basically non-standard and should in fact result in a network error.

**Comment by hillbrad** [comment]
Yes, should indicate via metadata that this is non-conformance affecting.

**Comment by mikewest** [comment]
I'm fairly certain that Chrome doesn't support the `jar:` scheme. I think it's fine to add the tests, but I don't really know what Chrome will do with it, and I hope we can just canonicalize whatever it does right now as "correct". :)
